addons/mail/static/src/core/common/mail_core_common_service.js

```diff
@@ -54,16 +49,13 @@ export class MailCoreCommon {
                 }
             });
             this.busService.subscribe("mail.message/notification_update", (payload) => {
-                payload.elements.map((message) => {
-                    this.store.Message.insert({
-                        ...message,
-                        body: markup(message.body),
-                        // implicit: failures are sent by the server at
-                        // initialization only if the current partner is
-                        // author of the message
-                        author: this.store.self,
-                    });
-                });
+                // implicit: failures are sent by the server at
+                // initialization only if the current partner is
+                // author of the message
```

Comment:
Note that is assumption is not correct, `mail.message/notification_update` it sent to both author of the message and the current user, which can be different.
I'm not sure why we need to assign the author like that (instead of receiving from the server), but it is just wrong.